src/bases/DataProviderBase.ts

```diff
@@ -55,6 +55,14 @@ abstract class DataProviderBase<T, O extends Options> {
 			if (this._data) {
 				observer.next(this._data);
 			}
+			return {
+				unsubscribe: () => {
```

Comment:
Looking [at the proposal](https://github.com/tc39/proposal-observable), this return is just supposed to be a cleanup function, versus an object.  Is this not the case with our shim?